celest/coordinates/astronomical_quantities.py

```diff
@@ -0,0 +1,602 @@
+    ----------
+    .. [Mee98d] Jean Meeus. Astronomical algorithms. 2nd ed. Willmann-Bell,
+       1998, pp. 147. isbn: 9780943396613.
+
+    Examples
+    --------
+    >>> julian = Quantity(2446895.5, u.jd2000)
+    >>> epsilon_0 = mean_obliquity(julian=julian)
+    """
+    t, = _calculate_raw_elapsed_jd_century_powers(julian, 1)
+    U = t / 100
+
+    result = (84381.448 - 4680.93 * U - 1.55 * U ** 2 + 1999.25 * U ** 3 -
+              51.38 * U ** 4 - 249.67 * U ** 5 - 39.05 * U ** 6 + 7.12 * U **
+              7 + 27.87 * U ** 8 + 5.79 * U ** 9 + 2.45 * U ** 10) / 3600
+
+    return Quantity(result, u.deg)
+
+
+def apparent_obliquity(julian: Quantity) -> Quantity:
+    """Return the ecliptic's apparent obliquity in the J2000 epoch.
+
+    Parameters
+    ----------
+    julian : Quantity
+        Quantity object containing date data in the JD2000 epoch.
+
+    Returns
+    -------
+    Quantity
+        Quantity object containing the ecliptic's apparent obliquity in degrees
+        and decimals.
+
+    See Also
+    --------
+    mean_obliquity :
+        Return the ecliptic's mean obliquity in the J2000 epoch.
+
+    Notes
+    -----
+    The methods to calculate the apparent obliquity of the ecliptic are given
+    in "Astronomical Algorithms" by Jean Meeus and are only valid for 10000
+    years on either side of J2000. [Mee98e]_
+
+    References
+    ----------
+    .. [Mee98e] Jean Meeus. Astronomical algorithms. 2nd ed. Willmann-Bell,
+       1998, pp. 147. isbn: 9780943396613.
+
+    Examples
+    --------
+    >>> julian = Quantity(2446895.5, u.jd2000)
+    >>> epsilon = apparent_obliquity(julian=julian)
+    """
+    average_obliquity = mean_obliquity(julian)
+    _, nutation_in_obliquity = nutation_components(julian)
+
+    return average_obliquity + nutation_in_obliquity
+
+
+def from_julian(julian: Quantity) -> Tuple:
+    """Return year, month, day from Julian times.
+
+    Parameters
+    ----------
+    julian : Quantity
+        Quantity object containing date data in the JD2000 epoch.
+
+    Returns
+    -------
+    Tuple
+        Returns a tuple, `(year, month, day)`, containing 1-D NumPy arrays
+        representing the year, month, and decimal days of the input times.
+
+    Notes
+    -----
+    Refer to page 63 of Astronomical Algorithms by Jean Meeus for a detailed
+    implementation of the algorithm. [Mee98f]_
+
+    References
+    ----------
+    .. [Mee98f] Jean Meeus. Astronomical algorithms. 2nd ed. Willmann-Bell,
+       1998, pp. 63. isbn: 9780943396613.
+
+    Examples
+    --------
+    >>> julian = Quantity(2436116.31, u.jd2000)
+    >>> apparent_obliquity(julian=julian)
+    (1957, 10, 4.81)
+    """
+
+    if isinstance(julian.to(u.jd2000), (float, int)):
+        jd = np.array([julian.to(u.jd2000)]) + 0.5
+    else:
+        jd = np.array(julian.to(u.jd2000)) + 0.5
+    a, f = jd.astype(int), jd % 1
+
+    ind = np.where(a >= 2291161)[0]
+    alpha = ((a[ind] - 1867216.25) / 36524.25).astype(int)
```

Comment:
I believe 36524.25 is a more accurate number of days in a Julian century. Consider updating the constants file and replacing this with a constant.